utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -1108,6 +1108,11 @@ def create_argument_parser():
            help='Enable experimental Swift differentiable programming language'
                 ' features.')
 
+    option('--enable-experimental-structural', toggle_true,
+           default=True,
```

Comment:
Why is this `True` and the `enable-experimental-structural` `0` above and `ENABLE_EXPERIMENTAL_STRUCTURAL` below `FALSE`?  Which ends up taking precedence?  Would it not be better for the option to be set uniformly?

Reply:
Thanks for noticing. We followed the precedent for related experimental language flags: `build-script` flag defaults to true, but CMake flag defaults to false.

---

What would you prefer? Defaulting both `build-script` and CMake flags to true?

We would like the flag to be true via invoking `build-script` so that the language features are tested by [ci.swift.org](http://ci.swift.org) on `main` and don't regress.

Reply:
Following the precedent is good.  I think that defaulting the CMake to `FALSE` and setting the value to true in `build-script` and in `build-presets.ini` is fine.  So the difference would be that the default is off, but all the CI configuration would enable it.  Additionally, this requires an additional change: utils/build-windows.bat will need to be adjusted to enable the configuration as well.

Reply:
I discussed with @compnerd offline.

---

Conclusion:
- For `build-script`: [the `--enable-experimental-{differentiable-programming,concurrency,structural}` options default to `true`](https://github.com/apple/swift/blob/f37f6944f9dc050981967e36facd1a3639a9c120/utils/build_swift/build_swift/driver_arguments.py#L1106-L1118). This allows these experimental language features to be tested by [ci.swift.org](http://ci.swift.org) on `main` so that they don't regress.

https://github.com/apple/swift/blob/f37f6944f9dc050981967e36facd1a3639a9c120/utils/build_swift/build_swift/driver_arguments.py#L1106-L1118

- For CMake: [the `ENABLE_EXPERIMENTAL_{DIFFERENTIABLE_PROGRAMMING,CONCURRENCY,STRUCTURAL}` flags default to false](https://github.com/apple/swift/blob/deb16ac3c4436d2ff72c82a38f3c762cb023d1af/CMakeLists.txt#L402-L408).

https://github.com/apple/swift/blob/deb16ac3c4436d2ff72c82a38f3c762cb023d1af/CMakeLists.txt#L402-L408

- `utils/build-presets.ini` does not have special values for these `build-script` experimental language feature flags, except the the `mixin_stdlib_minimal` preset which sets them all to disabled.

https://github.com/apple/swift/blob/f37f6944f9dc050981967e36facd1a3639a9c120/utils/build-presets.ini#L2363-L2366

- `utils/build-windows.bat`: the CMake experimental language flags are enabled. ~Two AutoDiff tests need to be fixed before `ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING` can be [enabled](https://github.com/apple/swift/pull/34865): TF-1197.

https://github.com/apple/swift/blob/f37f6944f9dc050981967e36facd1a3639a9c120/utils/build-windows.bat#L267-L268